src/theme/index.ts

```diff
@@ -1,8 +1,8 @@
 import defaultPalette from './styles';
 
-const BASE: number = 16;
+const fontSizes = [16, 24, 32, 48];
```

Comment:
inconsistent const naming conventions. Typically consts/statics are UPPER_SNAKE_CASE, but at least have them match imo

Reply:
Updated all of them in the theme file.